Modules/Utils.cs

```diff
@@ -172,6 +177,7 @@ public static bool HasTasks(GameData.PlayerInfo p, bool ForRecompute = true)
                     if (cSubRole == CustomRoles.Lovers)
                     {
                         //ラバーズがクルー陣営の場合タスクを付与しない
+                        // A Crew task win does not require Lovers tasks
```

Comment:
Not necessary as it is already written in Japanese.

Reply:
Ok